libs/components/popovers/src/lib/modules/dropdown/dropdown.component.scss

```diff
@@ -18,6 +16,13 @@
   --sky-override-context-menu-font-size: var(--sky-font-size-body-m);
 }
 
+@include compatMixins.sky-modern-overrides('.sky-dropdown-button-type-tab') {
+  --sky-override-tab-selected-background-color: var(--modern-color-transparent);
+  --sky-override-tab-selected-border-bottom-size: var(--modern-size-3);
+  --sky-override-tab-button-radius: 0;
+  --sky-override-tab-line-height: 1.8;
+}
+
 .sky-dropdown-button-type-tab {
   @include mixins.sky-btn-tab();
```

Comment:
Should we just move the contents of these mixins here? This is the last places these mixins are used.

Reply:
We could also then eliminate the import all together in this file.

Reply:
I can move those if we'd like--these are the default styles though. Did you want to change those?

Reply:
I don't necessarily want to change them - but we talked about removing the mixins. And moving the styles from the mixin here would help prepare us for that.

Reply:
there were 2 places actually (here and _buttons), so i moved them to both